pkg/kv/kvserver/replica_follower_read.go

```diff
@@ -73,10 +73,10 @@ func (r *Replica) canServeFollowerReadRLocked(ctx context.Context, ba *roachpb.B
 		return false
 	}
 
-	switch typ := repDesc.GetType(); typ {
```

Comment:
optional: Personally, I would leave this assignment to avoid retyping `repDesc.Type` below (here and in a few other cases in this PR), but it seems just a taste thing.

Reply:
Yeah, left it like this in a few places where we reuse the value quite a lot, but I didn't see much point in it here.